Replace purrr::cross2() with tidyr::expand_grid()

ramongss opened the pull request:
Fixes #24.

ramongss committed:
Replace purrr::cross2() with tidyr::expand_grid()

gvelasq commented:
Thank you. I will review in more detail in the coming weeks. Could you please update `NEWS.md`?

ramongss commented:
For sure, I'll add both PR in the `NEWS.md`.

ramongss committed:
Update NEWS.md

gvelasq committed:
`git merge origin/main`

Merge remote-tracking branch 'origin/main' into ramongss-replace-cross2-expand-grid

# Conflicts:
#	NEWS.md

gvelasq committed:
Fix imports

gvelasq committed:
Fix use of `tidyr::expand_grid()`
